### ISSUE TITLE
Some problems about running train.py

### ISSUE THREAD

**Issue by seven-sent** [microsoft/Deep3DFaceReconstruction#151]
Hello! Thank you for sharing your great project!

I tried to run the training code on Windows, but I got some problems.

` TypeError: Input 'filename' of 'ReadFile' Op has type float32 that does not match expected type of string.
`

Can I run train.py on Windows?

On the other hand, I tried to run the code on other computer using Linux. However, the computer doesn't seem to support CUDA9.0. That is, I cannot run the code with tensorflow 1.12. 

Are there any other higher versions of tensorflow available?
Or could you give me some advice?

**Comment by YuDeng** [comment]
Hi, our training code is not supported on Windows.
Our code can be used under other versions of tensorflow such as 1.13 or 1.14. You can choose other versions according to your own environment.

**Comment by seven-sent** [comment]
Thank you for your reply! I'll try again.
But I still have some questions.

If I install tensorflow 1.13 or 1.14 using pip, it means that I need to recompile the binary file (rasterize_triangles_kernel.so).
Is the method the same as  using conda?

> git clone https://github.com/google/tf_mesh_renderer.git
cd tf_mesh_renderer
git checkout ba27ea1798
git checkout master WORKSPACE
bazel test ...

**Comment by YuDeng** [comment]
Yes, you have to recompile the binary file. It should follow the same way as in the readme.

**Comment by seven-sent** [comment]
Thanks for your reply.
I created an environment with CUDA 10.0 and Cudnn 7.6.5, and installed tensorflow 1.13.1.
But some problems came up when I followed the readme to compile the binary file `rasterize_triangles_kernel.so`.

When I ran `bazel test ...`
the first test `//mesh_renderer/kernels:rasterize_triangles_impl_test ` passed and others failed.
However, I could not find the complied file `rasterize_triangles_kernel.so`.
I could not find the path `bazel-bin/mesh_renderer/kernels`.
Can you tell me what its absolute path is?

**Comment by YuDeng** [comment]
Well, can you try reset -D_GLIBCXX_USE_CXX11_ABI=0 and rebuild the Ops?

**Comment by seven-sent** [comment]
Thank you very much for your reply. I'm sorry to have been interrupting you.

I reset -D_GLIBCXX_USE_CXX11_ABI=0 and rebuilt the file. The error was solved, but another error occured.

`2021-04-20 12:50:46.278339: E tensorflow/stream_executor/cuda/cuda_blas.cc:698] failed to run cuBLAS routine cublasSgemm_v2: CUBLAS_STATUS_EXECUTION_FAILED`

Maybe I should try other version of tensorflow.
